docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -627,6 +1068,26 @@ Default: `UTC`
 
 Creates a filter that matches a given column to an exact value.
 
+*Expression syntax*
+[source,js]
+----
+exactly “state” value=”running”
+exactly “age” value=50 filterGroup=”group2”
+exactly column=“project” value=”beats”
+----
+
+*Code example*
+[source,text]
+----
+filters
+| exactly column=project value=elasticsearch
+| demodata
+| pointseries x=project y="mean(age)"
+| plot defaultStyle={seriesStyle bars=1}
+| render
+----
+The `exactly` filter here is added to existing filters retrieved by the `filters` function and further filters down the data to only have `”elasticsearch”` data. 
```

Comment:
It might be good to specific that this filter only applies to this element and shouldn't effect any other elements.